lib/modules/manager/mix/extract.ts

```diff
@@ -6,9 +6,12 @@ import type { PackageDependency, PackageFile } from '../types';
 
 const depSectionRegExp = regEx(/defp\s+deps.*do/g);
 const depMatchRegExp = regEx(
-  /{:(?<depName>\w+),\s*(?<datasource>[^:"]+)?:?\s*"(?<currentValue>[^"]+)",?\s*(?:organization: "(?<organization>.*)")?.*}/gm
+  /{:(?<app>\w+)(\s*,\s*"(?<requirement>[^"]+)")?(\s*,\s*(?<opts>[^}]+))?}/gm
 );
+const githubRegexp = regEx(/github:\s*"(?<value>[^"]+)"/);
+const organizationRegexp = regEx(/organization:\s*"(?<value>[^"]+)"/);
 const commentMatchRegExp = regEx(/#.*$/);
+const githubDataSourceId = 'github';
```

Comment:
We should use `GithubTagsDatasource` here instead.

Reply:
yes, will work. if there can be a git tag then set it as current value and don't skip, so renovate can update that tag to newer versions.

Reply:
Thanks, I applied your suggestions and also expanded to git repository handling - mix allows to set the repository either through the `git` or `github` option, and specify the thing to checkout either as `branch`, `tag` or `ref`.